### PR TITLE
build/configs: update defconfig for imxrt1050-evk tc_protected

### PR DESCRIPTION
**Pull request by sangam-swami**
Update defconfig changes to fix booting issue in tc_protected

Signed-off-by: Sangamanatha <sangam.swami@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d0929845579e2bbc135e5ab9bf71fa7a59561c3b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d0929845579e2bbc135e5ab9bf71fa7a59561c3b] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@sangam-swami Could you enable SDRAM?

**Comment by sunghan-chang** [comment]
@sangam-swami Could you update me?

**Comment by seinfra** [comment]
Target : [e393b42c7d59a6f4cf1fb0559977e71e22e432b3] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e393b42c7d59a6f4cf1fb0559977e71e22e432b3] - Code Rule Check (C++) OK.

**Comment by sangam-swami** [comment]
> @sangam-swami Could you enable SDRAM?
yes did

File: build/configs/imxrt1050-evk/tc-protected/defconfig [hunk 1]
```diff
@@ -175,12 +177,15 @@ CONFIG_IMXRT_LPSPI1=y
...
 # CONFIG_IMXRT_SNVS_HPRTC is not set
 # CONFIG_IMXRT_USDHC is not set
 # CONFIG_IMXRT_GPIO_IRQ is not set
 
+CONFIG_IMXRT_SEMC_SDRAM=y
+CONFIG_IMXRT_SDRAM_START=0x80000000
+CONFIG_IMXRT_SDRAM_SIZE=33554432
 #
 # Memory Configuration
 #
```
Comment (sunghan-chang) [inline build/configs/imxrt1050-evk/tc-protected/defconfig]: 1. You should change `CONFIG_IMXRT_OCRAM_PRIMARY` to `CONFIG_IMXRT_SDRAM_PRIMARY`.
2. Please don't modify this manually. You should modify this through menuconfig and then update this by copying .config.

Comment (sangam-swami) [reply build/configs/imxrt1050-evk/tc-protected/defconfig]: updated

**Comment by seinfra** [comment]
Target : [77013fa7f600aeaf26164513f60e10e4ddcb2b8b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [77013fa7f600aeaf26164513f60e10e4ddcb2b8b] -

**Pull request merged**